README.md

```diff
@@ -31,63 +42,153 @@ envs:
 
-If you want to use your step in your project's `bitrise.yml`:
+## Complex usage
 
-1. git push the step into it's repository
-2. reference it in your `bitrise.yml` with the `git::PUBLIC-GIT-CLONE-URL@BRANCH` step reference style:
+### Example task using the `testSearchQuery`
 
+```yml
+- git::https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests.git:
+   inputs:
+   - api_key: $DATADOG_API_KEY
+   - app_key: $DATADOG_APP_KEY
+   - test_search_query: 'tag:e2e-tests'
 ```
-- git::https://github.com/user/my-step.git@branch:
-   title: My step
+
+### Example task using the `testSearchQuery` and variable overrides
+
+```yml
+- git::https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests.git:
    inputs:
-   - my_input_1: "my value 1"
-   - my_input_2: "my value 2"
+   - api_key: $DATADOG_API_KEY
+   - app_key: $DATADOG_APP_KEY
+   - test_search_query: 'tag:e2e-tests'
+   - variables: |
+      START_URL=https://staging.website.com
+      PASSWORD=$(StagingPassword)
 ```
 
-You can find more examples of step reference styles
-in the [bitrise CLI repository](https://github.com/bitrise-io/bitrise/blob/master/_examples/tutorials/steps-and-workflows/bitrise.yml#L65).
+### Example task using a global configuration override with `configPath`
 
-## How to contribute to this Step
+This task overrides the path to the global `datadog-ci.config.json` file.
 
-1. Fork this repository
-2. `git clone` it
-3. Create a branch you'll work on
-4. To use/test the step just follow the **How to use this Step** section
-5. Do the changes you want to
-6. Run/test the step before sending your contribution
-  * You can also test the step in your `bitrise` project, either on your Mac or on [bitrise.io](https://www.bitrise.io)
-  * You just have to replace the step ID in your project's `bitrise.yml` with either a relative path, or with a git URL format
-  * (relative) path format: instead of `- original-step-id:` use `- path::./relative/path/of/script/on/your/Mac:`
-  * direct git URL format: instead of `- original-step-id:` use `- git::https://github.com/user/step.git@branch:`
-  * You can find more example of alternative step referencing at: https://github.com/bitrise-io/bitrise/blob/master/_examples/tutorials/steps-and-workflows/bitrise.yml
-7. Once you're done just commit your changes & create a Pull Request
+```yml
+- git::https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests.git:
+   inputs:
+   - api_key: $DATADOG_API_KEY
+   - app_key: $DATADOG_APP_KEY
+   - config_path: './synthetics-config.json'
+```
+
+For an example configuration file, see this [`global.config.json` file][B-1].
 
+### Example including all possible configurations
 
-## Share your own Step
+There shouldn't be any cases where you would need to use all of the configurations at the same time as for example `public_ids` and `test_search_query` are mutually exclusive. However having an example for the value of each is useful.
```

Comment:
No need to be too wordy about that, and I believe that in a perfect world the `public_ids` and `test_search_query` shouldn't be mutually exclusive. For now this is an implementation detail, which can change in datadog-ci 😉 

```suggestion
For reference, here is how a full configuration could look like:
```